README.md

```diff
@@ -1,8 +1,50 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C++, C#, C, JavaScript (Vue and ReactJS), MongoDB
+
+**Familiar:** .NET, Swift, Kotlin, Assembly language, Waterfall Methodology
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent to manage and insert resource data, such as CPU, RAM, disk usage into a PostgreSQL database hosted by Docker. The host usage information was collected through bash commands and automated using crontab. The data collected by the app can be returned as a result set by using SQL queries which also includes detecting crontab insertion failure using SQL aggregate functions to compare the number of entries within a span of five minutes. Other information, such as retrieving number of CPUs and amount of total memory can be used for data analytics when query the database. This app was developed and run within a CentOS7 environment.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/core_java)]:
+      
+  - Twitter App: In-progress
+  - JDBC App: Developed a Java application that uses Java Database Connectivity (JDBC) to insert and perform CRUD operations on data stored in a Docker-hosted PostgreSQL database. This project uses Java 8, RDBMS, JDBC, PSQL, Maven, Docker, DAO, and DTOs.
+  - Grep App: Implemented an application to simulate the Linux grep command using core Java, Maven, and Docker. It uses Java 8 lambdas and stream APIs to optimize the memory usage. This app is deployed on DockerHub and was developed/debugged using IntelliJ.
+
+
+## Highlighted Projects
+**Electronic Medical Records System**: Created a proof-of-concept EMR system for Otekha Health using ReactJS, Java RESTful server, Docker, MongoDB, MySQL, Redis, and HAPI-FHIR API. The development of this application followed an agile methodology and a Docker-built architecture. Synthea was used to generate sample patient FHIR data for testing purposes. MongoDB was used to hold the sample FHIR data retrieved from the RESTful server. MySQL was used for authentication within the app. Redis was used for access control (NGAC). The application allowed for users to customize the widgets that would be on their screen at once, including calendar, test results, medication, etc. Information could be added, removed, or modified within each component on the screen. The calendar application could be used to create a new event and bring the user to a new window where the corresponding patients' data would be displayed. The system was designed with the intention of being a customizable version of the OSCAR EMR system.
```

Comment:
**Electronic Medical Records System**: Created a proof-of-concept EMR system for Otekha Health using ReactJS, Java RESTful server, Docker, MongoDB, MySQL, Redis, and HAPI-FHIR API. The development of this application followed an agile methodology and a Docker-built architecture. Synthea was used to generate sample patient FHIR data for testing purposes. MongoDB was used to hold the sample FHIR data retrieved from the RESTful server. MySQL was used for authentication within the app. Redis was used for access control (NGAC). The application allowed users to customize the widgets on their screen at once, including calendar, test results, medication, etc. Information could be added, removed, or modified within each component on the screen. The calendar application could be used to create a new event and bring the user to a new window where the corresponding patients' data would be displayed. The system was designed with the intention of being a customizable version of the OSCAR EMR system.